schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
+    id integer AUTO_INCREMENT
```

Comment:
## [OPTIONAL]
* This is an optional suggestion but i think you can try it out
* I think making the id column as primary key as it gets a unique index for each primary key column that helps with faster access and also It uniquely identifies each row of a table 👍

Reply:
ok thanks, I'll do as you say